db/seeds/all.rb

```diff
@@ -0,0 +1,28 @@
+
+# Get the seed text from locale files
+
+codes = I18n.t('.enrollments.states.add_exemptions.code')
+desc  = I18n.t('.enrollments.states.add_exemptions.summary')
+
+unless(codes.is_a?(Hash) && desc.is_a?(Hash))
+  puts "ERROR - Expecting Exemptions seed data in locale key : '.enrollments.states.add_exemptions' + code & summary"
+  puts "GOT", codes.inspect, desc.inspect
+  Kernel.abort("ABORT : Missing locale information for seeding Exemptions")
+end
+
+FloodRiskEngine::Exemption.delete_all unless(Rails.env.production?)
+
```

Comment:
@reggieb  : agreed, changed to protect whole script. 

I believe prod seeds will go in a speciifc production file later